integration/containerd/shimv2/shimv2-tests.sh

```diff
@@ -25,9 +25,4 @@ if [[ "$ID" =~ ^opensuse.*$ ]]; then
 	exit
 fi
 
-if [ "$ID" != "centos" ]; then
```

Comment:
this looks like quite a big change/replace that is not called out in the commit - can you explain? Am I missing something?

Reply:
Hi @grahamwhaley Since the shimv2 testcases would fail randomly on "centos", and it was skipped on "centos", this fix tried to fix the random failure, thus we can drop this skip now.